WorkloadManagementSystem/Client/DIRACbenchmark.py

```diff
@@ -55,7 +55,7 @@ def singleDiracBenchmark(iterations=1, measuredCopies=None):
       start = os.times()
 
     # Now the iterations
-    for _j in xrange(n):
+    for _j in range(n):
```

Comment:
This is the benchmarking module, here performance is key. I would use `six.moves.range`